explainer.md

```diff
@@ -336,6 +325,62 @@ if(frameOfRef.bounds) {
 }
 ```
 
+### Mirroring
```

Comment:
This is in the advanced functionality section, actually. (Not always easy to tell with the sparse diffs)